src/args/arg.rs

```diff
@@ -3315,10 +3315,10 @@ impl<'a, 'b> Arg<'a, 'b> {
     /// Specifies that if the value is not passed in as an argument, that it should be retrieved
     /// from the environment if available. If it is not present in the environment, then default
     /// rules will apply.
-    pub fn from_env<K: AsRef<OsStr>>(mut self, name: K) -> Self {
+    pub fn env(mut self, name: &'a OsStr) -> Self {
```

Comment:
I think by convention, this should be `env_os` and there would also be another method `env` which would accept an `&str`.

Reply:
I notice in most of the other code, it's using OsStr::from_bytes, rather than OsStr::new(..), any particular reason?